docs/csharp/language-reference/keywords/where-generic-type-constraint.md

```diff
@@ -10,57 +10,68 @@ f1_keywords:
   - "whereconstraint_CSharpKeyword"
 helpviewer_keywords: 
   - "where (generic type constraint) [C#]"
-ms.assetid: d7aa871b-0714-416a-bab2-96f87ada4310
-caps.latest.revision: 10
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # where (generic type constraint) (C# Reference)
-In a generic type definition, the `where` clause is used to specify constraints on the types that can be used as arguments for a type parameter defined in a generic declaration. For example, you can declare a generic class, `MyGenericClass`, such that the type parameter `T` implements the <xref:System.IComparable%601> interface:  
-  
-```csharp  
-public class MyGenericClass<T> where T:IComparable { }  
-```  
-  
+
+In a generic type definition, the `where` clause is used to specify constraints on the types that can be used as arguments for a type parameter defined in a generic declaration. For example, you can declare a generic class, `MyGenericClass`, such that the type parameter `T` implements the <xref:System.IComparable%601> interface:
```

Comment:
Though the first sentence is from the current edition of the docs, can we rephrase it? *In a generic type definition...* sets scope directly to generic types, omitting methods and delegates (though, these two are mentioned at the end of the article). What about:
> The `where` clause is used to specify constraints on the types that can be used as arguments for a type parameter defined in a declaration of a generic type, method, or delegate.